.github/workflows/check-if-releasing-library.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: jwalton/gh-find-current-pr@v1
       - id: validation
         run: |
-          if [[ "${{ steps.PR.outputs.title }}" == "@storyblok/field-plugin-cli@"* ]]; then
+          if [[ "${{ steps.PR.outputs.title }}" == "@storyblok/field-plugin@"* ]]; then
```

Comment:
This is a bug fix.

What it does: "when releasing a library, don't test the templates. do it when you release the CLI"